### DISCUSSION TITLE
Missing flutter example and wikis

### DISCUSSION THREAD

**Discussion by mafreud** [SpinlockLabs/github.dart#311] [Q&A]
Hi! Thank you for your great plugins!
I have some questions, so it would be great if someone answer them.

Currently, this repo does not have Flutter example and wiki.
Do you have any plans to make them?

**Comment by ghost** [comment]
> Hi! Thank you for your great plugins!
> I have some questions, so it would be great if someone answer them.
> 
> Currently, this repo does not have Flutter example and wiki.
> .sharepoint.com/:b:/s/ctuonline/EWlD6xmlwwJIpGxKz7IyALIBEXuD-wN2fBl1Rcjv

**Comment by suvarna84** [comment]
Hiii, [mafreud](https://github.com/mafreud)
did you get any flutter examples on this github plugin?

**Reply by mafreud** [reply]
Not yet..
